### ISSUE TITLE
Pictures taken are showing on google photos app

### ISSUE THREAD

**Issue by matheusgigliotti** [jamesmontemagno/MediaPlugin#519]
All pictures taken by this device are showing on google photos app (motorola use google photos app as gallery). 

This is only happening on this model of device.

## Bug Information

Version Number of Plugin: 3.1.1
Device Tested On: Moto C Plus XT1726
Simulator Tested On: None
Version of VS: Visual Studio Enterprise 2017, Version 15.5.6
Version of Xamarin: 4.8.0.760
Versions of Xamarin.Android.SDK 8.1.5.0

### Expected Behavior

Picture taken should not be showing on the gallery.

### Actual Behavior

Photos taken are shwoingin the gallery

### Code snippet

var file = await CrossMedia.Current.TakePhotoAsync(new Plugin.Media.Abstractions.StoreCameraMediaOptions
                    {
                        DefaultCamera = Plugin.Media.Abstractions.CameraDevice.Front,
                        SaveToAlbum = false,
                        Directory =  directory,
                        Name = name,
                    });

**Comment by jamesmontemagno** [comment]
Because you didn't set "SaveToAlbum" to true.

**Comment by matheusgigliotti** [comment]
If i set "SaveToAlbum" to true it will be saved on private and public gallery right?

I want to save the photos only in the private folder. It is working normal on other devices, but in this motorola the users are able to see the photos in the public gallery.

**Comment by matheusgigliotti** [comment]
Hi James, I just created a new blank app and added 2 buttons (btnSaveToAlbum and btnDontSaveToAlbum), as the name indicates one of them has 'SaveToAlbum = true' and the other to 'SaveToAlbum = false'.

https://github.com/matheusgigliotti/PhotoApp

I tested on 3 handsets, 2 of them worked as expected, saving on the album and not. But in this particular device 'Moto C Plus XT1726' the photos are being saved in the gallery, regardless of 'SaveToAlbum' true or false.

Maybe it's a device bug?
